alterer-generator.cpp

```diff
@@ -48,15 +39,38 @@ alterer hairy;
 alterer stony;
 alterer murderer;
 
-alterer monster_alterer[13] = {weak, pacifist, dumb, blind, imaginary, legless,
+// Negative alterers
+alterer pacifist;
+alterer dumb;
+alterer blind;
+alterer imaginary;
+alterer legless;
+
+alterer monster_alterer[13] = {pacifist, dumb, blind, imaginary, legless,
     invisible, gigantic, three_headed, terrifying, hairy, stony, murderer, default_};
 
-alterer get_random_item_alterer(){//Pega um alterador aleatório da lista
-    int random = rand() % item_alterer.size();
+alterer get_random_item_alterer() {
+    int random = rand() % 9;
     return item_alterer[random];
 }
 
-alterer get_random_monster_alterer(){//Pega um alterador aleatório da lista
-    int random = rand() % monster_alterer.size();
+alterer get_random_monster_alterer() {
+    int random = rand() % 13;
     return monster_alterer[random];
+}
+
+void generate_generic_alterer(alterer &a) {
+    a.text = "text";
+    a.strength = 10;
```

Comment:
where these numbers come from? You gotta wrap them in well named constants

Reply:
this is a random number for temporary testing until we have defined the final values. On the new commit I specified